PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PaymentButton.kt

```diff
@@ -72,6 +72,27 @@ abstract class PaymentButton<C : PaymentButtonColor> @JvmOverloads constructor(
             prefixTextView.visibility = prefixTextVisibility
         }
 
+    /**
+     * Custom corner radius
+     */
+    var customCornerRadius: Float? = null
+        set(value) {
+            field = value
+
+            val cornerTreatment = if (value == 0.0f) {
```

Comment:
It's probably an edge case that a merchant would set `customCornerRadius` as null, but would we want to have a null check here and not change the corner treatment?